ports/libflac/vcpkg.json

```diff
@@ -19,6 +20,10 @@
     "asm": {
       "description": "Use any assembly optimization routines",
       "supports": "x86"
+    },
+    "stack-protector": {
+      "description": "Build with stack smashing protection",
```

Comment:
For my understanding we need to make it a default feature, to not disable the stack-protector for all targets.

Reply:
https://github.com/microsoft/vcpkg/pull/37155